site/docs/pages/getting-started.mdx

```diff
@@ -11,25 +11,48 @@ npm install @coinbase/onchainkit
 
 # Depending on the components or utilities you use,
 # you may end up utilizing any of those libraries.
-npm install viem@2.x react@18 react-dom@18
+npm install viem@2.x react@18 react-dom@18 @tanstack/react-query@5.x
```

Comment:
Let's add `@tanstack/react-query@5.x` as peer only in the identity docs, which is where the components are.

https://onchainkit.xyz/identity/introduction

so we can keep OnchainKit as modular as possible.